package.json

```diff
@@ -145,13 +143,6 @@
       "pre-push": "sh bin/validate_push"
     }
   },
-  "pnpm": {
-    "overrides": {
-      "vite-node": "2.0.1",
-      "vite": "5.4.8",
-      "vitest": "2.0.1"
-    }
-  },
```

Comment:
@KubesDavid we added this because Hisotire was not working, removing this breaks histoire, you can run it using `pnpm story:dev`.  Please take a look

Reply:
Histoire seems to be working on [Vite 6 support](https://github.com/histoire-dev/histoire/pull/770). Switching to the draft for now.